### ISSUE TITLE
cannot_log_to_file,"C:/temp/logs"

### ISSUE THREAD

**Issue by sebthom** [isindicic/RabbitMqPortable#1]
When starting RabbitMQ with a non-adminstrative user, it crashes with the following error on startup. How can I make rabbitmq not wanting to log to `C:/temp/logs`?

```erlang
BOOT FAILED
===========

Error description:
   {error,{cannot_log_to_file,"C:/temp/logs",{error,eisdir}}}

Log files (may contain more information):
   C:/temp/logs
   C:/apps/RABBIT~1/data/log/RABBIT~1.LOG

Stack trace:
   [{rabbit,ensure_working_log_handler,5,[{file,"src/rabbit.erl"},{line,818}]},
    {rabbit,ensure_working_log_handlers,0,
            [{file,"src/rabbit.erl"},{line,789}]},
    {rabbit,'-boot/0-fun-0-',0,[{file,"src/rabbit.erl"},{line,268}]},
    {rabbit,start_it,1,[{file,"src/rabbit.erl"},{line,403}]},
    {init,start_em,1,[{file,"init.erl"},{line,1076}]},
    {init,do_boot,3,[{file,"init.erl"},{line,784}]}]

{"init terminating in do_boot",{error,{cannot_log_to_file,"C:/temp/logs",{error,eisdir}}}}
```

**Comment by sebthom** [comment]
I found an environment variable defined on my corporate workstation called `LOGS` which points to `C:\temp\logs`. 
I wrapped the RabbitMqPortable.exe in a Batch file where I unset the LOGS variable first, then launch works. Maybe such special variables that could conflict with the portable rabbitmq instance could be unset automatically by the RabbitMqPortable.exe or the batch files found in rabbitmq_server-3.6.6/sbin.
